hugegraph-core/pom.xml

```diff
@@ -192,6 +192,24 @@
             <version>0.11.2</version>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-netty</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-protobuf</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-stub</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+        </dependency>
```

Comment:
do we include it in `commons-rpc` module?

Reply:
1、I compile protobuf with different version. and found that old file compliled with likely 2.6.1 version. this version released in Apr, 2015.  
https://mvnrepository.com/artifact/com.google.protobuf/protoc

2、if we add protobuf plugin. and generate file to src/protobuf/java path. this path will be ignored to check

3、Through the dependency check, found these dependencies already exist.

```
[INFO] com.baidu.hugegraph:hugegraph-rpc:jar:2.0.1:compile
[INFO] +- com.lmax:disruptor:jar:3.3.7:compile
[INFO] \- com.alipay.sofa:sofa-rpc-all:jar:5.7.6:compile
[INFO]    +- com.alipay.sofa:tracer-core:jar:3.0.8:compile
[INFO]    |  +- io.opentracing:opentracing-api:jar:0.22.0:compile
[INFO]    |  +- io.opentracing:opentracing-noop:jar:0.22.0:compile
[INFO]    |  +- io.opentracing:opentracing-mock:jar:0.22.0:compile
[INFO]    |  \- io.opentracing:opentracing-util:jar:0.22.0:compile
[INFO]    +- com.alipay.sofa.lookout:lookout-api:jar:1.4.1:compile
[INFO]    +- io.swagger:swagger-core:jar:1.5.18:compile
[INFO]    |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.3:compile
[INFO]    |  +- io.swagger:swagger-models:jar:1.5.18:compile
[INFO]    |  |  \- io.swagger:swagger-annotations:jar:1.5.18:compile
[INFO]    |  \- javax.validation:validation-api:jar:1.1.0.Final:compile
[INFO]    +- io.grpc:grpc-netty-shaded:jar:1.28.0:compile
[INFO]    |  \- io.grpc:grpc-core:jar:1.28.0:compile (version selected from constraint [1.28.0,1.28.0])
[INFO]    |     +- io.grpc:grpc-api:jar:1.28.0:compile (version selected from constraint [1.28.0,1.28.0])
[INFO]    |     |  \- io.grpc:grpc-context:jar:1.28.0:compile
[INFO]    |     +- com.google.code.gson:gson:jar:2.8.6:compile
[INFO]    |     +- com.google.android:annotations:jar:4.1.1.4:compile
[INFO]    |     \- io.perfmark:perfmark-api:jar:0.19.0:compile
[INFO]    +- io.grpc:grpc-protobuf:jar:1.6.1:provided
[INFO]    |  +- com.google.protobuf:protobuf-java-util:jar:3.3.1:provided
[INFO]    |  +- com.google.api.grpc:proto-google-common-protos:jar:0.1.9:provided
[INFO]    |  \- io.grpc:grpc-protobuf-lite:jar:1.6.1:provided
[INFO]    \- io.grpc:grpc-stub:jar:1.6.1:provided
```